docs/content/backup-restore/rman-backups.md

```diff
@@ -100,6 +100,83 @@ rman3-inst-opts   mydb            Physical      Instance         4
 
 Once the backup phase changed to `Succeeded`, the physical backup creation is complete and ready to use.
 
+## Scheduling regular backups
+
+El Carro can manage schedules and retention periods for RMAN backups.
+
+### Prepare the backup schedule manifest
+
+To schedule a backup, create a manifest of type `BackupSchedule` with the following fields:
+
+* backupSpec: the parameters for the backup itself, as for a [one-off backup](#preparing-a-backup-cr-manifest)
```

Comment:
I like that you provided a thorough descriptions of these fields. We should do more of that in our docs.

Reply:
In this case I'm trying to follow the existing structure and style of the doc.  The canonical location for this type of documentation is actually the CRD: https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/blob/77fdd40aa416d8b76f813aa0e598ee731ab05286/oracle/config/crd/bases/oracle.db.anthosapis.com_backupschedules.yaml.